standards/cheat_sheet.md

```diff
@@ -193,6 +211,8 @@ Payment schemes, which describe the "protocols" of payment for services/resource
 |---|---|---|---|---|
 |**`golem.com.scheme`**|`String`|Offer||Scheme of payments made for computing resources consumed. |
 |**`golem.com.scheme.payu.interval_sec`**|`Number`|Offer||For "pay-as-you-use" payment scheme, indicates interval of invoices issued during the service usage. |
+|**`golem.com.scheme.payu.debit-notes.interval-sec?`**|`Number`|Demand/Offer|Negotiable, Experimental|For "pay-as-you-use" payment scheme, indicates interval of Debit Notes issued during the service usage. |
```

Comment:
I see more examples containing `_` within the name like one in the line above. You committed marking them deprecated purposely (eg. not to change too much)?

Reply:
So, for usage counters I chose to 'deprecate' the old ones and create new ones, as it is just the name change (no change in sematics). 

For these particular "pay-as-you-use" properties, the pre-existing one (`golem.com.scheme.payu.interval_sec`) is not changig, and it may become superseded by `golem.com.scheme.payu.debit-notes.interval-sec?` with different sematics, but for now the new property is marked as `Experimental`. Once we are happy with new properties, we'll make them 'official' and deprecate the old one. 

Does this make sense?